caos.ansible_roles/roles/docker_install/tasks/install-docker-SLES.yaml

```diff
@@ -11,10 +11,6 @@
   delay: 60
   when: ansible_distribution_version not in suse_versions_wo_docker
 
-- name: install docker in old version
-  ansible.builtin.include_tasks: "install-docker-SLES_old.yaml"
-  when: ansible_distribution_version in suse_versions_wo_docker
-
 - name: install docker-compose (manual)
```

Comment:
`suse_versions_wo_docker` as 12.3 and 12.4. Is it correct to remove this? ☝️ 
```yaml
suse_versions_wo_docker:
  - "12.3"
  - "12.4"

```

Reply:
Done in https://github.com/newrelic-experimental/caos-ansible-roles/pull/71/commits/699efc8f31cd9f2c5e2abbe76cbd9b0a7f5ea8a7!